spikingjelly/activation_based/triton_kernel/flex_sn_inductor/hop.py

```
@@ -203,18 +618,930 @@ def eager_scan_final_state(
+            NestedUserFunctionVariable,
+            UserFunctionVariable,
+        )
+        from torch._dynamo.variables.higher_order_ops import (
+            TorchHigherOrderOperatorVariable,
+            make_attr,
+            speculate_subgraph,
+        )
+        from torch._dynamo.variables.tensor import TensorVariable
+    except (ImportError, ModuleNotFoundError, AttributeError):
+        return
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)